cmd/kpod/umount.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	umountCommand = cli.Command{
+		Name:        "umount",
+		Aliases:     []string{"unmount"},
+		Usage:       "Unmount a working container's root filesystem",
+		Description: "Unmounts a working container's root filesystem",
```

Comment:
Are we using the "working container" terminology in kpod, or should it just be "container" here?